src/tools/tidy/src/deps.rs

```diff
@@ -885,21 +885,11 @@ fn check_license_exceptions(
                     ));
                 }
                 Some(pkg_license) => {
-<<<<<<< HEAD
                     if licenses.iter().all(|license| *license != pkg_license.as_str()) {
-                        println!(
-                            "dependency exception `{name}` license in workspace `{workspace}` has changed"
-                        );
-                        println!("    previously `{licenses:?}` now `{pkg_license}`");
-                        println!("    update EXCEPTIONS for the new license");
-                        *bad = true;
-=======
-                    if pkg_license.as_str() != *license {
                         check.error(format!(r#"dependency exception `{name}` license in workspace `{workspace}` has changed
-    previously `{license}` now `{pkg_license}`
+    previously `{licenses:?}` now `{pkg_license}`
```

Comment:
I am unclear why we're deviating from upstream here?

Reply:
[we allow multiple deps with different licenses](https://github.com/ferrocene/ferrocene/commit/79ede149db3c11a4e200eadab2d8ef6fef2fbc26)

note: we will not need to carry this change if we move to having same Cargo.lock with upstream

Reply:
Ok!